modules/microshift-firewall-apply-settings.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-firewall-applying-settings_{context}"]
+== Applying firewall settings
+After you have finished configuring, run the following command to restart the firewall and apply settings:
+
+[source,terminal]
+----
+$ sudo firewall-cmd --reload
+[source,terminal]
+----
+
+[id="microshift-firewall-verifying-settings_{context}"]
+== Verifying settings
+After you have restarted the firewall, you can verify your settings by listing them.
+
+To verify rules added in the default public zone, such as the ports related rules, run the following command:
```

Comment:
These would be single step procedure, so we'd use * at the beginning. Also, note the minor removal of [source,terminal] in the command. 

```
* To verify rules added in the default public zone, such as the ports related rules, run the following command:
+
[source,terminal]
----
$ sudo firewall-cmd --list-all
----

* To verify rules added in the trusted zone, such as IP-range related rules, run the following command:
+
[source,terminal]
----
$ sudo firewall-cmd --zone=trusted --list-all
----
```